### PR TITLE
Have other node derive address so it can see issuance/reissuance

### PR DESCRIPTION
**Pull request by wintercooled**
It's not realistic to have the issuer (or reissuer) share an address with someone who wants to view issuance details - more likely the person needing details will look up a txid. This change shows how to derive an address form tha tx and import that so the details are known by the wallet.

### CODE REVIEW

File: elements-code-tutorial/easy-run-code.md [hunk 1]
```diff
@@ -230,9 +230,10 @@ e1-cli generatetoaddress 1 $ADDRGEN1
 sleep 10
 e2-cli listissuances
 
-IADDR=$(e1-cli gettransaction $ITXID | jq -r '.details[0].address')
-
-e2-cli importaddress $IADDR
+ISSUE_RAW_TX=$(e2-cli getrawtransaction $ITXID 1)
```
Comment (stevenroose) [inline elements-code-tutorial/easy-run-code.md:233]: As opposed to how it was before, this requires node2 to have the txindex enabled. While before you asked the address from node1 who has it in its wallet. Why is this change required perse?

File: elements-code-tutorial/easy-run-code.md [hunk 2]
```diff
@@ -230,9 +230,10 @@ e1-cli generatetoaddress 1 $ADDRGEN1
...
+ISSUE_VOUTS=$(echo $ISSUE_RAW_TX | jq -r '.vout')
+VOUT_ADDRESS_ISSUE=$(echo $ISSUE_VOUTS | jq -r '.[0].scriptPubKey.addresses[0]')
+e2-cli importaddress $VOUT_ADDRESS_ISSUE
```
Comment (stevenroose) [inline elements-code-tutorial/easy-run-code.md:236]: I don't know if the other effects of importing the address might be annoying here for the user. This might mean that fundrawtransaction (perhaps with include_watchonly set to true) to include UTXOs sent to that address.

File: elements-code-tutorial/easy-run-code.md [hunk 3]
```diff
@@ -301,9 +302,11 @@ sleep 10
 e1-cli listissuances
 
 RITXID=$(echo $RISSUE | jq -r '.txid')
-RIADDR=$(e2-cli gettransaction $RITXID | jq -r '.details[0].address')
+REISSUE_RAW_TX=$(e1-cli getrawtransaction $RITXID 1)
```
Comment (stevenroose) [inline elements-code-tutorial/easy-run-code.md:305]: Same as above.

File: elements-code-tutorial/easy-run-code.md (hunk 1 above)
Comment (wintercooled) [reply elements-code-tutorial/easy-run-code.md:233]: It was for a scenario where a second node wants to see an issuance. So e1 issues an asset and sends the reissuance token to e2 - e2 cannot reissue because it is not aware of the issuance so an address needs importing from the issuance tx. e2's user can look up the tx on the block explorer's assets list page and doesn't need to know the address issued to but yes, this approach required tx_index=1.

Perhaps I can leave it as it was and just add the 'use a txid to get an address' as a commented out option?

File: elements-code-tutorial/easy-run-code.md (hunk 2 above)
Comment (wintercooled) [reply elements-code-tutorial/easy-run-code.md:236]: What other way is there for another node to know about an issuance a node made? Do you know?

File: elements-code-tutorial/easy-run-code.md (hunk 1 above)
Comment (stevenroose) [reply elements-code-tutorial/easy-run-code.md:233]: Hmm, depends on your scenario. If the scenario is "user1 sends one of the issuance tx addresses to user2", then use `e1-cli gettransaction`. If the scenario is "user1 sends the issuance txid and user2 looks for an address, then yeah `e2-cli getrawtransaction` makes sense. Perhaps a note "if txindex is not enabled, an address can be found by entering the txid in a block explorer", but it's good as is.

File: elements-code-tutorial/easy-run-code.md (hunk 2 above)
Comment (stevenroose) [reply elements-code-tutorial/easy-run-code.md:236]: Tbh I don't think there is a way to do that explicitly. So yeah this seems like the best guess. If you think this is a feature worth having, feel free to open an issue on the elements github. I'm not entirely sure how hard it would be.

File: elements-code-tutorial/easy-run-code.md (hunk 1 above)
Comment (wintercooled) [reply elements-code-tutorial/easy-run-code.md]: have added mention of need for index=1 in conf file for this to work and reinstated use of previous code and offered this as an option instead.
